geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisKey.java

```diff
@@ -21,23 +21,26 @@
 import java.io.DataInput;
 import java.io.DataOutput;
 import java.io.IOException;
+import java.util.Arrays;
 
+import org.apache.geode.DataSerializer;
 import org.apache.geode.internal.serialization.DataSerializableFixedID;
 import org.apache.geode.internal.serialization.DeserializationContext;
 import org.apache.geode.internal.serialization.KnownVersion;
 import org.apache.geode.internal.serialization.SerializationContext;
 import org.apache.geode.redis.internal.executor.cluster.CRC16;
 import org.apache.geode.redis.internal.executor.cluster.RedisPartitionResolver;
+import org.apache.geode.redis.internal.netty.Coder;
 
-public class RedisKey extends ByteArrayWrapper implements DataSerializableFixedID {
+public class RedisKey implements DataSerializableFixedID, Comparable<RedisKey> {
```

Comment:
Have you found that we still need to implement `Comparable`?

Reply:
I think it's likely that we don't. I'll remove this and see if anything breaks in pre-checkin.